codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AwsServiceIdIntegration.java

```diff
@@ -54,7 +58,13 @@ public SymbolProvider decorateSymbolProvider(
     }
 
     private static Symbol updateServiceSymbol(Symbol symbol, String serviceId) {
-        String name = serviceId.replace(" ", "") + "Client";
+        String name = String.join("", Arrays.asList(
```

Comment:
nit: we use 8 spaces for continuing a line
minor: this could be a stream with a `joining` collector:

```java
String name = serviceId.split(" ").stream()
        .map(StringUtils::capitalize)
        .collect(Collectors.joining("")) + "Client";
```

Reply:
Thanks! Replaced with the following code in commit [`493c47d` (#1379)](https://github.com/aws/aws-sdk-js-v3/pull/1379/commits/493c47de47997a1fe6175d238453823e56d707db)

```java
        String name = Arrays.asList(serviceId.split(" ")).stream()
                .map(StringUtils::capitalize)
                .collect(Collectors.joining("")) + "Client";
```